tests/sentry/integrations/slack/test_message_builder.py

```diff
@@ -194,14 +194,16 @@ def test_build_performance_issue(self):
         ):
             event = perf_event_manager.save(self.project.id)
         event = event.for_group(event.groups[0])
-        attachments = SlackIssuesMessageBuilder(event.group, event).build()
+        with self.feature("organizations:performance-issues"):
+            attachments = SlackIssuesMessageBuilder(event.group).build()  # do not pass the event
```

Comment:
You'll always end up having a latest event here because we wrote one to the snuba, and so `get_latest_event` should succeed. What you could try doing to repro is to manually create a performance group via `self.create_group` and try passing that instead.

I think I know why `self.group.get_latest_event()` is returning nothing. In theory this should never happen, because there should always be an event associated with the group. However, there's a race condition between writing the event to snuba (which we're querying to get the latest event) and running this message logic. If this runs before we manage to actually write the event, then we'll end up with no event here.

This only happens with perf issues and will be solved by https://getsentry.atlassian.net/browse/ISP-10 when I actually get time to work on it.

Reply:
Ah okay, for now I am passing the group to `get_color` and hard coding it to info for perf issues but left a comment that it's a temp fix and updated the test as you suggested. I don't know if this is the most pressing thing to merge but I was notified of it being wrong ... idk I'd be okay with leaving it as is or throwing in this mildly hacky fix for now and trying my best to remember to take it out later (though it doesn't hurt to leave it in).